docs/hippocampus_model.md

```diff
@@ -0,0 +1,35 @@
+# Trained model for hippocampal segmentation
+## Purpose
+This documentation describes how to use our pre-trained model to segment the left and right hippocampi from brain MRI scans. The model was trained on 998 MRI scan + segmentation pairs from the [ADNI](https://adni.loni.usc.edu/) dataset. This data is publicly available via their website, but users must sign a Data Use Agreement in order to gain access. We do not provide access to the data. The following description assumes the user has their own dataset to evaluate/ retrain the model on.
+
+## Terms of use
+Please note that this model is intended for research purposes only. You are responsible for the performance, the necessary testing, and if needed any regulatory clearance for any of the models produced by this toolbox.
+
+---
+
+## Usage
+The following instructions assume you have completed the preceding setup steps in the [InnerEye README](https://github.com/microsoft/InnerEye-DeepLearning/), in particular, [Setting up Azure Machine Learning](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/setting_up_aml.md).
+
+### Create an Azure ML Dataset
+To evaluate this model on your own data, you will first need to register an [Azure ML Dataset](https://docs.microsoft.com/en-us/azure/machine-learning/v1/how-to-create-register-datasets). You can follow the instructions in the InnerEye repo for [creating datasets](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/creating_dataset.md) in order to do this.
+
+## Downloading the model
+The saved weights from the trained Hippocampus model can be downloaded along with the source code used to train it from [our github releases page](https://github.com/microsoft/hi-ml/releases).
+
+### Registering a model in Azure ML
+To evaluate the model in Azure ML, you must first [resgister an Azure ML Model](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.model.model?view=azure-ml-py#remarks). To register the Hippocampus model in your AML Workspace, unpack the source code downloaded in the previous step and follow the instructions to run InnerEye's [move_model.py](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/move_model.md) script in the `-a=upload` direction.
```

Comment:
I was expecting the second link to take me to that script. I would either remove the Python extension or replace the target URL with a link to the code.

Reply:
Also, I suggest removing the mention to `-a` in the text. I will suggest some more changes for readability in the snippet below.

Reply:
The link to the documentation? That is intentional as that docu contains the instructions to follow

Reply:
Ok but then I would remove the `.py` suffix from the link text, I think it's slightly misleading. Maybe it could be something like

```markdown
unpack the source code downloaded in the previous step and follow the [instructions to upload models to Azure ML](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/move_model.md).
```